third_party/tpch-dbgen/tpch_dbgen.h

```diff
@@ -12,6 +12,7 @@ typedef struct
 {
   long      weight;
   char     *text;
+  char      len;
```

Comment:
using char for lengths makes me nervous, using size_t shouldn't hurt us in any way, right?